amundsen_application/static/js/components/common/Table/index.tsx

```diff
@@ -91,9 +133,11 @@ const Table: React.FC<TableProps> = ({
     isLoading = false,
     numLoadingBlocks = DEFAULT_LOADING_ITEMS,
     rowHeight = DEFAULT_ROW_HEIGHT,
+    expandRow = null,
```

Comment:
Is it expected that this isn't and issue for the `strictNullChecks` rule, given that this is defined above as an optional `Function`?

Reply:
It didn't complain!
Not sure if there is a better option though... happy to try other solutions!